tools/releaseBuild/yaml/releaseBuild.yml

```
@@ -68,6 +68,10 @@ stages:
             LINUX_BUILDNAME: 'alpine'
           UbuntuArm64:
             LINUX_BUILDNAME: 'ubuntu.16.04-arm64'
+          AlpineArm64:
+            LINUX_BUILDNAME: 'alpine-arm64'
+          AlpineArm64:
+            LINUX_BUILDNAME: 'alpine-arm'
```

```suggestion
          AlpineArm:
            LINUX_BUILDNAME: 'alpine-arm'
          AlpineArm64:
            LINUX_BUILDNAME: 'alpine-arm64'
```

Extra important! The job name was duplicated and was probably going to fail (they were both named `AlpineArm64`.

---

Good catch!